components/builder-jobsrv/habitat/default.toml

```diff
@@ -16,8 +16,4 @@ connection_timeout_sec = 3600
 
 [archive]
 backend = "local"
-key = ""
```

Comment:
The `default.toml` should be considered an "interface" and expose all configurable pieces. I would include these bits with a sensible default if possible and just a blank string when it's something required. In the future I'd really love for us to investigate writing something that allows us to describe our interface much better than TOML itself so we can validate config on the Habitat level instead of bogusly configuring a service